BBConst.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+
+/* ======== Rollback ======== */
+constexpr s32 MAX_ROLLBACK_FRAMES = 5;
+constexpr s32 FRAME_DELAY = 2;
+static_assert(FRAME_DELAY >= 1);
+static_assert(FRAME_DELAY + MAX_ROLLBACK_FRAMES >= 6); // minimum frames of "compensation"
+constexpr bool ROLLBACK_IMPL = true;
+
+/* ======== General Netplay ======== */
+// number of max FrameData's to keep in the (remote) queue
+constexpr s32 FRAMEDATA_MAX_QUEUE_SIZE = 15; 
+static_assert(FRAMEDATA_MAX_QUEUE_SIZE > MAX_ROLLBACK_FRAMES);
+constexpr s32 GAME_START_FRAME = 0;
+// before this frame we basically use delay-based netcode to ensure things are reasonably synced up before doing rollback stuff
+constexpr s32 GAME_FULL_START_FRAME = 100;
+constexpr s32 MAX_REMOTE_PLAYERS = 3;
+constexpr s32 MAX_NUM_PLAYERS = 4;
+constexpr s32 BRAWLBACK_PORT = 7779;
+
+/* ======== Timesync ======== */
+// update ping display every X frames
+constexpr s32 PING_DISPLAY_INTERVAL = 30;
+// check clock desynchronization every X frames
+constexpr s32 ONLINE_LOCKSTEP_INTERVAL = 30;
+constexpr s32 TIMESYNC_MAX_US_OFFSET = 10000; // 60% of a frame
+
+/* ======== Debug Toggles ======== */
+//#define SYNCLOG
+//#define RANDOM_INPUTS
+
+/* ======== Utils ======== */
+#define MS_IN_FRAME (1000 / 60)
```

Comment:
I think these defines should be either constexpr vars or functions

Reply:
Also, from my understanding a lot of these constants are only needed in -asm repo OR in dolphin. So I don't see why we should add them to the -common repo.

e.g. PING_DISPLAY_INTERVAL is only needed in dolphin right?

Reply:
I will more comprehensively adjust the defines, but the push I just put up addresses some of this.